Add content from community doc PRs (920, 924, 929) (#162)
rancher/harvester-product-docs: versions/v1.7/modules/en/pages/upgrades/upgrades.adoc

In production environments, upgrading clusters via the UI is recommended.
====

== Customize node upgrades

{harvester-product-name} upgrades involve several defined phases. A key phase is node upgrades, during which the operating system and the underlying Kubernetes distribution (RKE2) are upgraded on each node sequentially and autonomously.

> You have the option to pause automatic upgrades on specific nodes, which is useful when manual maintenance or verification tasks must be performed. Following the completion of these tasks, you must explicitly instruct {harvester-product-name} to resume the upgrade on the target nodes.

Reviewer: ```suggestion
You have the option to pause automatic upgrades on specific nodes, useful for manual maintenance or verification tasks. Following the completion of these tasks, you must explicitly instruct {harvester-product-name} to resume the upgrade on the target nodes.
```
